nvflare/app_opt/flower/flower_job.py

```diff
@@ -69,17 +69,13 @@ def __init__(
             client_shutdown_timeout (float, optional): Timeout for client shutdown. Defaults to 5.0 seconds.
             stream_metrics (bool, optional): Whether to stream metrics from Flower client to Flare
             analytics_receiver (AnalyticsReceiver, optional): the AnalyticsReceiver to use to process received metrics.
-            client_api_type (str, optional): Client API type, can choose from EX_PROCESS_API and IN_PROCESS_API
+            extra_env (dict, optional): optional extra env variables to be passed to Flower client
         """
         if not os.path.isdir(flower_content):
             raise ValueError(f"{flower_content} is not a valid directory")
 
         super().__init__(name=name, min_clients=min_clients, mandatory_clients=mandatory_clients)
 
-        if client_api_type not in ["EX_PROCESS_API", "IN_PROCESS_API"]:
-            raise ValueError("Invalid client api type.")
-        os.environ["CLIENT_API_TYPE"] = client_api_type
```

Comment:
Just to clarify, why is this no longer needed? If CLIENT_API_TYPE is not set I believe it defaults to "IN_PROCESS_API" now

Reply:
yes, we decided not to have this one hard-coded here.
Instead, users will pass all the extra env they need.
That way it also works with the situations outside of Job API (for example POC, production)